### PR TITLE
Avail docs

### PR DESCRIPTION
**Pull request by deepakgudla**
Avail DA Documentation

## Summary by CodeRabbit

- **New Features**
  - Added a new sidebar link titled "Avail" that links to a new tutorial on deploying a GM world rollup on Avail's data availability (DA) network.
  - Introduced a detailed tutorial covering the setup and deployment of an Avail light node on the Turing testnet.

### CODE REVIEW

**Review by MSevey — APPROVED** [review]

**Comment by MSevey** [comment]
pr preview expected to fail, lgtm 👍🏻

File: tutorials/avail-da.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+
+## 🌞 Introduction {#introduction}
+
+Avail DA offers scalable data availability that underpins the Avail ecosystem and ensures instantaneous and reliable data integrity, enabling rollups to grow, through the use of cutting-edge zero knowledge and KZG Polynomial commitments.
+
+This tutorial serves as a comprehensive guide for deploying your GM world rollup on Avail's data availability (DA) network.
+
+Before proceeding, ensure that you have completed the [GM world rollup](/tutorials/gm-world) tutorial, which covers setting up a local sovereign gm-world rollup and connecting it to a local (mock) DA node.
+
+## 🪶 Running an avail light node
```
Comment (jcstein) [inline tutorials/avail-da.md:11]:
```suggestion
## 🪶 Running an Avail light node
```

capital Avail?

File: tutorials/avail-da.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+
+Before you can start your rollup node, you need to initiate, sync, and possibly fund a light node on Turing testnet which is the test network of Avail
```
Comment (jcstein) [inline tutorials/avail-da.md:13]:
```suggestion
Before you can start your rollup node, you need to initiate, sync, and possibly fund a light node on Turing Testnet which is the test network of Avail
```

File: tutorials/avail-da.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+
+- [Local Development](https://github.com/rollkit/avail-da/blob/main/README.md)
+- [Turing Testnet](https://docs.availproject.org/docs/networks)
+
+### 🚀 Using Turing Testnet
+
+- To fund your wallet address for using Turing Testnet: get AVAIL tokens from [Faucet]((https://faucet.avail.tools/))
```
Comment (jcstein) [inline tutorials/avail-da.md:20]:
```suggestion
- To fund your wallet address for using Turing Testnet: get AVAIL tokens from [the faucet]((https://faucet.avail.tools/))
```

File: tutorials/avail-da.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+This tutorial serves as a comprehensive guide for deploying your GM world rollup on Avail's data availability (DA) network.
+
+Before proceeding, ensure that you have completed the [GM world rollup](/tutorials/gm-world) tutorial, which covers setting up a local sovereign gm-world rollup and connecting it to a local (mock) DA node.
+
+## 🪶 Running an avail light node
+
+Before you can start your rollup node, you need to initiate, sync, and possibly fund a light node on Turing testnet which is the test network of Avail
+
+- [Local Development](https://github.com/rollkit/avail-da/blob/main/README.md)
```
Comment (jcstein) [inline tutorials/avail-da.md:15]:
```suggestion
- [Local development](https://github.com/rollkit/avail-da/blob/main/README.md)
```

File: tutorials/avail-da.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+- [Turing Testnet](https://docs.availproject.org/docs/networks)
+
+### 🚀 Using Turing Testnet
+
+- To fund your wallet address for using Turing Testnet: get AVAIL tokens from [Faucet]((https://faucet.avail.tools/))
+- Paste your mnemonic in the `identity.toml` file by creating a `identity.toml` with the following command:
+```touch identity.toml```
```
Comment (jcstein) [inline tutorials/avail-da.md:22]:
```suggestion
```bash
touch identity.toml
```
```

Comment (jcstein) [inline tutorials/avail-da.md:22]:
```suggestion
`touch identity.toml`
```

File: tutorials/avail-da.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+Example:
+```bash
```
Comment (jcstein) [inline tutorials/avail-da.md:24]: - [ ] add spaces around code blocks for linting, in all instances in this document

Comment (jcstein) [inline tutorials/avail-da.md:24]:
```suggestion

```bash
```

File: tutorials/avail-da.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+avail_secret_uri = '<paste your mnemonic here>'
+```
```
Comment (jcstein) [inline tutorials/avail-da.md:26]:
```suggestion
```

```

File: tutorials/avail-da.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+- Running just an Avail light node is enough for Turing testnet. Run the Avail light node using the following command
```
Comment (jcstein) [inline tutorials/avail-da.md:27]:
```suggestion
- Running just an Avail light node is enough for Turing Testnet. Run the Avail light node using the following command
```

File: tutorials/avail-da.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+echo -e "\n Your DA_BLOCK_HEIGHT is $DA_BLOCK_HEIGHT \n"
+```
+
+You will see the output like this:
+
+```bash
+ Your DA_BLOCK_HEIGHT is 35
+```
+
+## 🔥 Running your rollup connected to an avail light node
```
Comment (jcstein) [inline tutorials/avail-da.md:86]:
```suggestion
## 🔥 Running your rollup connected to an Avail light node
```

File: tutorials/avail-da.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+
+```bash
+    gmd start \
+    --rollkit.aggregator \
+    --rollkit.da_address="grpc://localhost:3000" \   
+    --rollkit.da_start_height $DA_BLOCK_HEIGHT \
+    --minimum-gas-prices="0.1stake"
+```
+
+Now, the rollup is running and posting blocks (aggregated in batches) to avail. You can view your rollup by finding your account on [Turing testnet](https://avail-turing.subscan.io/)
```
Comment (jcstein) [inline tutorials/avail-da.md:98]:
```suggestion
Now, the rollup is running and posting blocks (aggregated in batches) to Avail. You can view your rollup by finding your account on [Turing testnet](https://avail-turing.subscan.io/)
```

File: tutorials/avail-da.md [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+Example:
+```bash
+avail_secret_uri = '<paste your mnemonic here>'
+```
+- Running just an Avail light node is enough for Turing testnet. Run the Avail light node using the following command
+```bash
+cargo run --release -- --network turing --app-id 1 --clean --identity identity.toml
+```
+
+If you want to sync avail light node with your desired block number, you can add the following config in your `config.yaml` file from [here](https://github.com/availproject/avail-light/blob/main/config.yaml.template)
```
Comment (jcstein) [inline tutorials/avail-da.md:32]:
```suggestion
If you want to sync Avail light node with your desired block number, you can add the following config in your `config.yaml` file from [here](https://github.com/availproject/avail-light/blob/main/config.yaml.template)
```

File: tutorials/avail-da.md [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+
+## 🏗️ Building your rollup
+
+Now we need to rebuild our rollup by simply running the existing `init.sh` script:
+
+```bash
+cd $HOME/gm && bash init.sh
+```
+
+This process creates a new `$HOME/.gm` directory and a new `gmd` binary. Next, we need to connect our rollup to the running avail light node.
```
Comment (jcstein) [inline tutorials/avail-da.md:60]:
```suggestion
This process creates a new `$HOME/.gm` directory and a new `gmd` binary. Next, we need to connect our rollup to the running Avail light node.
```

File: tutorials/avail-da.md [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+
+## 🛠️ Configuring flags for DA
+
+Now we're prepared to initiate our rollup and establish a connection with the avail light node. The `gmd start` command requires two DA configuration flags:
```
Comment (jcstein) [inline tutorials/avail-da.md:64]:
```suggestion
Now we're prepared to initiate our rollup and establish a connection with the Avail light node. The `gmd start` command requires two DA configuration flags:
```

File: tutorials/avail-da.md [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+
+Now, the rollup is running and posting blocks (aggregated in batches) to avail. You can view your rollup by finding your account on [Turing testnet](https://avail-turing.subscan.io/)
+
+::: info
+For details on configuring gas prices specifically for the DA network, see our [DA Network Gas Price Guide](/guides/gas-price). This is separate from the `--minimum-gas-prices="0.025stake"` setting, which is used for rollup network operations.
+:::
+
+## 🎉 Next steps
+
+Congratulations! You've built a local rollup that posts to avail's testnets as well as locally. Well done! Now, go forth and build something great! Good luck!
```
Comment (jcstein) [inline tutorials/avail-da.md:106]:
```suggestion
Congratulations! You've built a local rollup that posts to Avail's testnets as well as locally. Well done! Now, go forth and build something great! Good luck!
```

**Review by jcstein — CHANGES_REQUESTED** [review]
made suggestions mostly on capitalization and some linting

otherwise LGTM after those are resolved

**Comment by MSevey** [comment]
@coderabbitai resolve

**Comment by MSevey** [comment]
@coderabbitai ignore

File: tutorials/avail-da.md [hunk 15]
```diff
@@ -0,0 +1,109 @@
...
+- To fund your wallet address for using Turing Testnet: get AVAIL tokens from [the faucet]((https://faucet.avail.tools/))
+- Paste your mnemonic in the `identity.toml` file by creating a `identity.toml` with the following command:
+`touch identity.toml`
+Example:
+
+```bash
+avail_secret_uri = '<paste your mnemonic here>'
+```
+
+Running just an Avail light node is enough for Turing testnet. Run the Avail light node using the following command
```
Comment (jcstein) [inline tutorials/avail-da.md:29]:
```suggestion
Running just an Avail light node is enough for Turing testnet. Run the Avail light node using the following command

```

**Review by jcstein — APPROVED** [review]
utACK

**Pull request merged**